spec/samples/bad_whitespace.lua

```diff
@@ -0,0 +1,22 @@
+-- Examples of whitespace formatting violations
+
+function trailing_whitespace_in_code()
+   return "This is awful" 
+end
+
+function trailing_whitespace_in_comment()
+   -- Less awful, but... 
+   return "Still bad"
+end
+
+function whitespace_only_lines()
+ 
+	
+		
+   
+   return "Lost in space"
+end
+
+function inconsistent_indentation()
+ 	return "Don't do this"
+end
```

Comment:
Thanks! I had tried `local` functions, but overlooked the obvious solution of returning a "module" table.